crank/DihedralScanner.py

```diff
@@ -434,8 +434,8 @@ def finish(self):
 
     def draw_ascii_image(self):
         """ Return a string with ASCII colors showing current running status """
-        if not hasattr(self, 'grid_energies') or not hasattr(self, 'running_job_path_info'):
```

Comment:
Thanks for pointing this out. I think the implementation switched from using `self.running_job_path_info` to using `self.opt_queue` at some point, but the check hasn't been updated.

Instead of removing the check for `running_job_path_info`, could you please replace it with the opt_queue?
So here line 437 and 438 will become
```
        if not hasattr(self, 'grid_energies') or not hasattr(self, 'opt_queue'):
            return "draw_ascii_image failed: grid_energies or opt_queue not available"
```